ang/civicase-base.ang.php

```diff
@@ -37,11 +37,18 @@
 
 /**
  * Expose settings.
+ *
+ * The default case category is taken from URL first,
+ * or uses `case` as the default.
  */
 function expose_settings(&$options) {
+  $urlCaseCategory = CRM_Utils_Request::retrieveValue('case_type_category', 'String');
```

Comment:
The case category is also being fetched at the top of this file. Let's use that variable instead.